apps/src/netsim/NetSimVizElement.js

```diff
@@ -151,6 +151,11 @@ NetSimVizElement.prototype.stopAllAnimation = function () {
  */
 NetSimVizElement.prototype.tweenToPosition = function (newX, newY, duration,
     tweenFunction) {
+  // Don't accept new animation commands if we've already been killed
+  if (this.isDying() || this.isDead()) {
+    return;
```

Comment:
Sometimes an element could be killed and then sent to the background immediately afterward, which would result in new tweens getting attached to it, leaving it in a 'dying' state until they finished which could be inappropriately long.  Really, we don't want to start any new animation on a node while it's dying.